deployments/charts/kuma/values.yaml

```diff
@@ -699,6 +702,17 @@ experimental:
     # -- Path where compiled eBPF programs which will be installed can be found
     programsSourcePath: /kuma/ebpf
 
+# -- Postgres settings for universal control plane on k8s
+postgres:
+  # -- Postgres hostname
+  host:
+  # -- Postgres port
+  port: "5432"
+  # -- Postgres user
+  user:
+  # -- Postgres database
+  db:
```

Comment:
I think we need to also expose TLS settings.

Would be great to have short doc how to run this on universal.
I can imagine we also need `password` to the DB. How to provide it.

Reply:
side question, do you think I should move this to "controlPlane.postgres"?

Reply:
I was thinking about the same thing. No strong opinion here. Technically, now only the control plane uses postgres, but this may or may not change in the future.